src/status_im/ui/components/copyable_text.cljs

```diff
@@ -67,7 +67,7 @@
     [react/text {:style {:typography :main-medium :line-height 20 :font-size 14}}
      (i18n/label :sharing-copied-to-clipboard)]]])
 
-(defview copyable-text-view [{:keys [label copied-text]} content]
+(defview copyable-text-view [{:keys [label copied-text container-style]} content]
```

Comment:
I know it was already there but can you remove the defview here, a regular defn is enough, you might just need to define `:reagent-render` to put what was in letsubs in a let binding outside of the rendering fn. If you need me to explain more in details please ping. Here is an example of what I mean https://github.com/status-im/status-react/blob/ba112a765bda13770064e11a95b340c135954dbc/src/status_im/ui/screens/signing/views.cljs#L248 (not that in the params of the reagent-render fn, you might want to repeat the params of the defn if you want the view to react on it, that was not the case in the example)

Reply:
I think I see what you mean. I always went for `defview` instead of `reagent/create-class` as I thought `defview` along with `letsubs` was a `status-react` specific helper designed to replace having to `create-class`. And I wondered also why there were some places where `reagent/create-class` is still employed.

I will change the implementation - will ping you if I have questions.

Reply:
done